ts/configs/metrics.yaml

```diff
@@ -1,34 +1,62 @@
 dimensions:
-  - &model_name "model_name"
+  - &model_name "ModelName"
   - &level "Level"
+  - &device_id "device_id"
 
 ts_metrics:
   counter:
-    - name: CounterTsMetricExample
-      unit: ms
-      dimensions: [*model_name, *level]
+    - name: Requests2XX
+      unit: count
+      dimensions: [*level]
+    - name: Requests4XX
+      unit: count
+      dimensions: [*level]
+    - name: Requests5XX
+      unit: count
+      dimensions: [*level]
   gauge:
-    - name: GaugeTsMetricExample
+    - name: QueueTime
       unit: ms
-      dimensions: [*model_name, *level]
-  histogram:
-    - name: HistogramTsMetricExample
+      dimensions: [*level]
+    - name: WorkerThreadTime
       unit: ms
-      dimensions: [*model_name, *level]
+      dimensions: [*level]
+    - name: CPUUtilization
+      unit: percent
+      dimensions: [*level]
+    - name: MemoryUsed
+      unit: MB
+      dimensions: [*level]
+    - name: MemoryAvailable
+      unit: MB
+      dimensions: [*level]
+    - name: MemoryUtilization
+      unit: percent
+      dimensions: [*level]
+    - name: DiskUsage
+      unit: GB
+      dimensions: [*level]
+    - name: DiskUtilization
+      unit: percent
+      dimensions: [*level]
+    - name: DiskAvailable
+      unit: GB
+      dimensions: [*level]
+    - name: GPUMemoryUtilization
+      unit: percent
+      dimensions: [*level, *device_id]
+    - name: GPUMemoryUsed
+      unit: MB
+      dimensions: [*level, *device_id]
+    - name: GPUUtilization
+      unit: percent
+      dimensions: [*level, *device_id]
 
 model_metrics:
```

Comment:
Add comment in the file.
dimension "hostname" is automatically added for model metrics at backend.